glog_backend.go

```diff
@@ -89,6 +91,14 @@ func RegisterBackend() <-chan Event {
 	return c
 }
 
+// clearBackends gets rid of all backends
+func clearBackends() {
```

Comment:
Looks like this is only used in tests. Perhaps it should move to a a test lib?

Reply:
Good point, although I'd like to try and avoid needing this at all by refactoring the backend code a bit (testing this is a nightmare and still doesn't quite work).

Reply:
Done. Refactoring will be a bit of a longer process, unfortunately.